client/js/widgets/widget.js

```diff
@@ -1300,6 +1300,52 @@ export class Widget extends StateManaged {
         }
       }
 
+      if(a.func == 'SCORE') {
+        setDefaults(a, { mode: 'set', property: 'score', seats: null, round: null, value: null});
+        if([ 'set', 'inc', 'dec' ].indexOf(a.mode) == -1) {
+          problems.push(`Warning: Mode ${a.mode} interpreted as set.`);
+          a.mode = 'set'
+        }
+        
+        if(a.value === null)
+          a.value = a.mode=='set' ? 0 : 1;
+        if(isNaN(parseFloat(a.value))) {
+          problems.push(`value ${a.value} must be a number, assuming 0.`);
+          a.value = 0;
+        }
+        a.value = parseFloat(a.value);
```

Comment:
I think we need to alow that the value could also be a property of the seat.

so in the game I was making I aws hoping to get the values from a counter. I did set up that this value in inherited by the seat. So it would be great to just put "text" in value.

It does work similarly if I set the score property in the scoreboard to be "text" I get the totals equal to the counter values:

![image](https://user-images.githubusercontent.com/73476656/207124719-d4a0d7a0-024f-4e26-939e-27bd92239fd0.png)

Reply:
What do you mean that "the value could also be a property of the seat?"  The values that go in the scoreboard can be any property of scoreboard (defaults to score).  Do you mean the total?  The totals are available as a read-only property.  You shouldn't be changing the totals.  Change the individual rounds values.

Reply:
I'm also not clear as to what is being asked for here.

Reply:
> {
>       "func": "SCORE",
>       "seats": [
>         "SEAT1",
>         "SEAT2",
>         "SEAT3",
>         "SEAT4",
>         "SEAT5",
>         "SEAT6"
>       ],
>       "property": "score",
>       "value": "text",
>       "mode": "set",
>       "round": null
>     }

I was hoping we could make that this format would add the value in "text" to the property "score" for each seat.
so in my example, the seat would inherit the value "text" from the counter. SCORE would therefore effectively save the value in the counter to the scoreboard

it seems to me that it could be compatible because value only accepts numbers at the moment.

Reply:
But most likely what you are doing with this is setting an _array_.  Which index are you trying to set with this?  If you aren't using an array and just using an overall total, then just use SET.  If you are working with an array of values (like using rounds), then the entry format is much more complicated.  Roger originally wrote the code that way.  It was so difficult to use programatically we discussed solutions and he changed it to what we have now.

Reply:
Why not use `textChangeRoutine`?

I also feel like it's weird to have counters _and_ a scoreboard.